src/utils/credit_card_util.js

```diff
@@ -23,12 +27,19 @@ CreditCardUtil.isCreditCardNumberValid = function (creditCardNumber) {
 };
 
 CreditCardUtil.isCreditCardExpirationDateValid = function (cardExpirationMonth, cardExpirationYear) {
-    return NUMBER_REGEX.test(cardExpirationMonth) &&
-        NUMBER_REGEX.test(cardExpirationYear) &&
-        Number(cardExpirationMonth) >= 1 &&
-        Number(cardExpirationMonth) <= 12 &&
-        Number(cardExpirationYear) >= 2016 &&
-        Number(cardExpirationYear) <= 2100;
+    if (
+        !NUMBER_REGEX.test(cardExpirationMonth) ||
+        NUMBER_REGEX.test(cardExpirationYear)
+    ) return false; // check if input is number
+
+    const expMonth = Number(cardExpirationMonth);
+    const expYear = Number(cardExpirationYear);
+    return (
+        expYear >= currentYear &&
+        expYear <= 2100 &&
```

Comment:
@imajindev could you mind changing into something like this
```suggestion
        expYear <= currentYear + 100
```